lang/llvm-3.4/Portfile

```diff
@@ -77,15 +75,12 @@ if {${subport} eq "llvm-${llvm_version}"} {
 }
 
 if {${os.platform} eq "darwin" && ${os.major} < 11 && ${cxx_stdlib} eq "libc++"} {
-    configure.cxx_stdlib    libstdc++
+    revision                    [ expr ${revision} + 1 ]
```

Comment:
Expressions should be brace-quoted. Having a different revision depending on the platform is a bit problematic anyway though.

Reply:
Let bump whole revision when.